huxley/settings/logging.py

```diff
@@ -1,12 +1,14 @@
 # Copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
 # Use of this source code is governed by a BSD License (see LICENSE).
 
+from .main import DEBUG
```

Comment:
This may not work if the local settings overrides the value of `DEBUG`. That is to say, I believe this value is always going to be `True`. Not sure how to get around that, I'll think about it.